src/status_im/contexts/profile/settings/style.cljs

```diff
@@ -6,8 +6,8 @@
   {:padding-top      inset
    :background-color (colors/resolve-color customization-color theme 40)})
 
-(def footer-container
-  {:margin-top         8
-   :margin-bottom      50
-   :padding-horizontal 20
-   :padding-vertical   12})
+(defn footer-container
+  [bottom]
+  {:padding-horizontal 20
+   :padding-top        20
+   :padding-bottom     (- 78 bottom)})
```

Comment:
- Its 77 in design.
- I think we should not subtract this bottom. 77 is used to make sure log out don't overlap jump-to button.
 So, jump-to button position will be (21(bottom padding) + 44 (jump-to button)  = 65

(So if we remove this bottom, bottom padding become = 77-21(as per design) = 56.

which is lower than, jump-to position.

Reply:
Here the idea is that it's at least 78 (or 77, whatever). If the bottom inset is set, like on iPhones, then it will be lower which is fine because the bottom inset is not part of the calculation anyway.

Reply:
@Parveshdhull https://github.com/status-im/status-mobile/pull/19318#issuecomment-2009695657